composer.json

```diff
@@ -16,19 +16,19 @@
     }
   ],
   "require": {
-    "php": ">= 7.2",
+    "php": ">= 7.4",
     "ext-json": "*",
-    "guzzlehttp/psr7": "^1.4|^2.0",
-    "nette/utils": "^3.1.0"
+    "guzzlehttp/psr7": "^1.4 || ^2.0",
+    "nette/utils": "^3.1"
   },
   "require-dev": {
-    "nette/di": "^3.0.1",
-    "nette/tester": "~2.3.1",
-    "contributte/guzzlette": "^3.0.0",
-    "nette/caching": "^3.0.0",
-    "ninjify/qa": "^0.12.0",
-    "ninjify/nunjuck": "^0.3.0",
-    "mockery/mockery": "^1.3.0",
+    "contributte/guzzlette": "^3.0",
```

Comment:
Mohl bys tam prosim vratit tu formu x.y.z? Ja to mam radeji.

Reply:
Z toho co jsem pochopil tak forma MAJOR.MINOR.PATCH je omezující pro ostatní kdo si stáhne tento balíček protože mu omezíš závislosti. Viz [composer manual](https://getcomposer.org/doc/articles/versions.md#caret-version-range-)

> For pre-1.0 versions it also acts with safety in mind and treats ^0.3 as >=0.3.0 <0.4.0 and ^0.0.3 as >=0.0.3 <0.0.4.

Reply:
Jaj, já mám manuál na monitoru odřádkovaný tak že jsem si nespojil pre-1.0. Takže to platí jen pro verze menší jak 1.0.0.

Reply:
Navíc si myslím, že je to takto sjednocený, už předtím tam bylo 
```
    "phpstan/phpstan": "^1.0",
    "phpstan/phpstan-deprecation-rules": "^1.0",
    "phpstan/phpstan-nette": "^1.0",
    "phpstan/phpstan-strict-rules": "^1.0"
```